lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,65 @@
+"""
+This script automates the copying of the default keymap into your own keymap.
+"""
+import os
+import sys
+
+from milc import cli
+from qmk.path import keymap, check_directories, create_keymap_directory
+
+
+# TODO(orngshdw): add debugging code if needed:
+#   @cli.argument('--debug', dest='debug', action='store_true', help='specify to turn on debugging messages')
+@cli.entrypoint('Create a new keymap.c from the default keyboard of your choosing')
+def main(cli):
+
+    # get keyboard
+    keyboard = input("Keyboard Name: ")
+    # get username
+    username = input("New Username: ")
+
+    # generate keymap paths
+    kb_path = os.path.abspath(keymap(keyboard))
+    print(kb_path)
+    keymap_path_default = kb_path + "/default"
+    keymap_path = kb_path + "/" + username
+
+    # check directories
+    check_directories(kb_path, keymap_path_default)
+    check_directories(keymap_path, exist=False)
+
+    # create user directory with default keymap files
+    create_keymap_directory(keymap_path_default, keymap_path)
+
+    # end message to user
+    print("%s keymap directory created in: %s\n" % (
```

Comment:
You should use `cli.log.info` here (and down below) instead of `print()`, and bring them up into 1 line.

Reply:
just for clarification, did you want all 4 print statements to be consolidated to one line or each of the print statements to be one line?

Reply:
In this case yes. Generally we want to split things like that into multiple lines when they get to be too complex, and these are all simple.